main/py3-meld3/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Peter Bui <pnutzh4x0r@gmail.com>
+# Maintainer:
+pkgname=py3-meld3
+_pkgname=meld3
+pkgver=1.0.2
+pkgrel=2
+pkgdesc="An HTML/XML templating engine"
+url="https://github.com/supervisor/meld3"
+arch="noarch"
+license="Repoze"
+depends="python3"
+makedepends="python3-dev"
```

Comment:
dependencies for testing can be added as

 checkdepends="py3-pytest py3-pytest-cov"


But requires moving py3-meld3 and supervisor to testing

Reply:
@maxice8 
> But requires moving py3-meld3 and supervisor to testing

I'm not sure what you are trying to say here.

Reply:
The dependencies live in testing and for check() to work they would need to be moved to main. An alternative would be to move supervisor and the check depends to community.

Reply:
@TBK Thanks, that cleared things up.